lib/webmock/http_lib_adapters/typhoeus_hydra_adapter.rb

```diff
@@ -172,7 +172,7 @@ def self.request_hash(request_signature)
                 request.execute_headers_callbacks(response)
               end
               if request.respond_to?(:streaming?) && request.streaming?
-                response.options[:response_body] = ""
+                response.options[:response_body] = String.new
```

Comment:
It's a corner case, using `"".dup` can reduce code size while maintaining behavioral compatibility.

```console
$ ruby -e 'p "".encoding'
#<Encoding:UTF-8>
$ ruby -e 'p "".dup.encoding'
#<Encoding:UTF-8>
$ ruby -e 'p String.new.encoding'
#<Encoding:BINARY (ASCII-8BIT)>
```

```suggestion
                response.options[:response_body] = "".dup
```

Reply:
Thanks for your review & feedback @koic !